src/api/labels.test.js

```diff
@@ -0,0 +1,93 @@
+import { checkLabel, saveLabel, saveName } from './labels'
+const KEY = 'labels'
+
+beforeEach(() => {
+  localStorage.clear()
+})
+
+function setupStorage() {
+  localStorage.__STORE__[KEY] = JSON.stringify({
+    '0x36940f34a2ec6afe46b7db53e6611470cd76c4f5999209a04a670682e2c33f75':
```

Comment:
Do you want to put these as constant at the top of the test file?
It's a bit hard to see which hash are identical and which is not. Same for `blahblah` (you can put into `label` constant or something).